lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -629,6 +629,7 @@ import qualified Cardano.Wallet.Primitive.Types.UTxOIndex as UTxOIndex
 import qualified Cardano.Wallet.Primitive.Types.UTxOSelection as UTxOSelection
 import qualified Cardano.Wallet.Primitive.Types.UTxOStatistics as UTxOStatistics
 import qualified Cardano.Wallet.Write.Tx as WriteTx
+import qualified Cardano.Wallet.Write.Tx as Write.Tx
```

Comment:
Could we please stick to `WriteTx` in this PR?

On the concrete topic itself: While I see how one could argue the ". -> camelCase" conversion is confusing, I think it makes sense as "one thing", and it doesn't seem as an alien concept in the wallet:

```
$ ag "as ([A-Z][a-z]+)([A-Z][a-z]+)"
[...]
import qualified Cardano.Wallet.DB.Store.Meta.Model as TxMetaStore
import qualified Data.List.NonEmpty as NonEmptyList
import qualified Cardano.Wallet.Api.Types as ApiTypes
```